frontend/src/components/common/vertical-scroll/vertical-scroll.tsx

```diff
@@ -69,7 +71,7 @@ const LeftArrow: FC<PropsLeftRightArrow> = ({ isFollowingOrBrowse = false }): Re
       isRight={false}
       isFollowingOrBrowse={isFollowingOrBrowse}
     >
-      {'<'}
+      <Left height={20} width={20} color="var(--brand-green-color)" />
```

Comment:
why you chnage arrow button icon? i have a another pr to fix it https://github.com/BinaryStudioAcademy/bsa-2022-streamlet/pull/275

Reply:
oops, maybe QA miss something